engines/dfc_provider/spec/controllers/dfc_provider/api/catalog_items_controller_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
-require 'spec_helper'
+require 'spec_helper.rb'
+require Rails.root.join('engines/dfc_provider/spec/spec_helper.rb')
```

Comment:
Couldn't find a better way to load engine-specific spec helper. And I need the ´spec_helper´ above as it is setting the ´Rails´ object.

Reply:
I dont think this must be in spec_helper.
You could just put the reset pref code inside a normal helper like PreferencesHelper and you would only need to do:
include PreferencesHelper
and then call a method in it for example reset_dfc_preferences

Reply:
Actually it is called ´spec_helper´ by some convention but it is more a spec setup. It is something we need even before the class is loaded. I mean I will need to call it everywhere, so to set it up for once was making more sense to me. It is done this way on the OFN main repo, https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/spec_helper.rb#L164.